operator/internal/validation/openshift/alertingrule.go

```diff
@@ -14,24 +14,33 @@ import (
 func AlertingRuleValidator(_ context.Context, alertingRule *lokiv1.AlertingRule) field.ErrorList {
 	var allErrs field.ErrorList
 
+	enabled, fieldErr := tenantIDValidationEnabled(alertingRule.Annotations)
```

Comment:
I think the result variable name can be improved:

```suggestion
	validateTenantIDs, fieldErr := tenantIDValidationEnabled(alertingRule.Annotations)
```